addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -99,7 +95,11 @@ $o-wsale-products-layout-grid-gutter-width: min($grid-gutter-width / 2, $o-wsale
     }
 
     .o_wsale_filmstip_container {
-        &:hover {
+        &.o_browser_not_chromium {
+            scrollbar-color: currentColor $light;
+        }
+
+        &:not(.o_browser_not_chromium):hover {
```

Comment:
It's a pity that scrollbar customization doesn't work on Firefox :(.

However, it should work on Safari. But since you're only targeting Chrome (`:not(.o_browser_not_chromium)`), the hover state doesn't work and you always get a too-small scrollbar on Apple's browser.

Could you also detect Safari and update the selector ?